src/core/AutomationPattern.cpp

```diff
@@ -548,8 +760,9 @@ void AutomationPattern::saveSettings( QDomDocument & _doc, QDomElement & _this )
 						it != m_timeMap.end(); ++it )
 	{
 		QDomElement element = _doc.createElement( "time" );
-		element.setAttribute( "pos", it.key() );
-		element.setAttribute( "value", it.value() );
+		element.setAttribute("pos", POS(it));
+		element.setAttribute("inValue", INVAL(it));
```

Comment:
This will make every automation pattern unusable in old versions of LMMS, right?

Reply:
Correct, old ones are upgraded though. Do you think I should use `value` for the `inValue` XML? It's less accurate but might make it usable on older versions if you think it's worth it. I didn't think of that because I thought only the other way around was necessary (being able to open older project files).